### DISCUSSION TITLE
How to join zerotier network after creation

### DISCUSSION THREAD

**Discussion by paktek123** [sinamics/ztnet#429] [Q&A] [answered]
Hi,

I have deployed ztnet using the docker compose example and everything is running as expected. I might be misunderstanding something but when I run `zerotier-cli join <networkid>` it would try to join the network controller on my.zerotier.com rather than the one I hosted. How do I join the network on the ztnet with my custom URL?

Please let me know if the above doesn't make sense.

**Comment by sinamics** [comment, marked as answer]
To join a network on ztnet instead of the default one on my.zerotier.com, you need to create a new network in ztnet. This will generate a unique network ID that distinguishes your network from all others, including those on my.zerotier.com.

Once you have the network ID, use the following command to connect your node to the network:

```bash
zerotier-cli join <networkid>
```

Replace **networkid** with the unique ID of the network you created in your ztnet controller. This command will connect your node to the network managed by ztnet.

**Reply by paktek123** [reply]
In my case, I ran the join command but the Network Member didn't show up immediately , so I came back after an hour and it showed up. I would recommend to wait a little bit. Thank you for the quick answer.

**Reply by dcorwin8222** [reply]
This delay has gotten me a few times. I noticed my node was originally "Relayed" but then quickly changed to "direct". I don't know if this has something to do with the delay on the initial connection.
